src/cmd/policy.go

```diff
@@ -84,6 +93,45 @@ func RegoFuncReadFile(ctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
 	return nil, nil
 }
 
+func RegoFuncGetGithubRepo(ctx rego.BuiltinContext, a, b *ast.Term) (*ast.Term, error) {
+
+	var org, repo string
+	if err := ast.As(a.Value, &org); err != nil {
+		return nil, err
+	} else if ast.As(b.Value, &repo); err != nil {
```

Comment:
this shouldn't be an else?

Reply:
🤷 That's how they have it in their example.  I think it works as an `else if` since they're just checking for errors when they store the value in each var but I could also see it working as just a separate if block.

Reply:
ya lets just make it 2 if as thats confusing if it really does work.  As it reads like the 2nd if will get skipped

Reply:
Done :-)